src/sentry/migrations/0256_create_docintegration_table.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 2.2.24 on 2021-12-08 18:23
+
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.jsonfield
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0255_delete_code_mappings_with_no_integration"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="DocIntegration",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
+                ("slug", models.CharField(max_length=64, unique=True)),
+                ("author", models.CharField(max_length=255)),
+                ("description", models.TextField()),
+                ("url", models.URLField()),
+                ("popularity", models.PositiveSmallIntegerField(default=1, null=True)),
+                ("is_draft", models.NullBooleanField(default=True)),
+                ("metadata", sentry.db.models.fields.jsonfield.JSONField(null=True)),
```

Comment:
Do you need any datetimes for when doc integrations were added or updated?

Reply:
No, I don't think so. This is purely an internal way for us to list integrations that other companies have made with us in the integration directory.